banana-stand/public/js/common/common-controllers.module.js

```diff
@@ -3,4 +3,4 @@
 var ng = require('angular');
 
 module.exports = ng.module('bst.common.controllers', [])
-    .controller('MasterController', require('./controllers/master.controller'));
+    .controller('MasterController', require('./controllers/master.controller.js'));
```

Comment:
I renamed the file and told Webstorm to update any references to it. I'll update it in #3